.github/workflows/rust.yml

```diff
@@ -25,12 +25,6 @@ jobs:
         config:
           [
             # Different DB cases:
-            {
```

Comment:
Hey why it's dropping Redis 6 tests ?

Reply:
the commit message is pretty clear - 
"Redis 6 doesn't support SSUBCSRIBE, and so fails on cluster-async-pubsub tests."

we can add a check for DB version in the test, but since redis 6 is soon out of support, I'm fine with removing the tests.

Reply:
decided to keep testing redis 6, and try to opt out of using ssubscribe when testing those servers.